vehicle/CVehicleModelInfo.hpp

```diff
@@ -111,7 +111,9 @@ class CVehicleModelInfo : public CBaseModelInfo
     float m_min_seat_height; //0x0554
     char pad_0558[40]; //0x0558
     uint32_t m_is_jetski; //0x0580
-    char pad_0584[28]; //0x0584
-}; //Size: 0x05A0
-static_assert(sizeof(CVehicleModelInfo) == 0x5A0);
+    char pad_0584[7]; //0x0584
+    uint16_t m_ability_flag; // 0x058B
+    char pad_058D[26]; // 0x058D
+}; //Size: 0x05A7
+static_assert(sizeof(CVehicleModelInfo) == 0x5A7);
```

Comment:
Best to keep the size as-is, unless you've verified from the constructor of the CVehicleModelInfo class how large the struct is.

I know functionality wise, it doesn't change much, but having it be accurate with what it's actual size is helps when mapping new fields.